Sming/Makefile-windows.mk

```diff
@@ -9,7 +9,7 @@ SDK_BASE	?= $(ESP_HOME)/ESP8266_SDK
 SDK_TOOLS	 ?= $(ESP_HOME)/utils
 
 # Other tools mappings
-ESPTOOL		 ?= $(SDK_TOOLS)/esptool.exe
+ESPTOOL		 ?= $(SDK_TOOLS)/ESP8266/esptool.exe
```

Comment:
Smuggling changes in PRs is not good! The change above has nothing to do with the main purpose of that PR.